protos/quote.proto

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+syntax = "proto3";
+
+message Quote {
+    enum Status {
+        OPEN = 0;
+        CLOSED = 1;
+    }
+
+    // ID of the quote
+    string quote_id = 1;
+
+    // Ticker symbol of the organization the quote is on bahalf of
+    string organization = 2;
```

Comment:
Can we name this `organization_id` as well? To keep it consistent across protos

Reply:
Right. We should make sure to reflect that in the transactions and DB as well.

Reply:
Still renaming this?